controls/tabview/styling/header-itemstyle-selector.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Header ItemStyle Selector
+page_title: .NET MAUI TabView Documentation - Header ItemStyle Selector
+description: Learn how to apply a style selector to the header items of the Telerik UI for .NET MAUI TabView.
+slug: tabview-header-itemstyle-selector
+tags: tabview, maui, dotnet, header, styleselector
+position: 3
+---
+
+# .NET MAUI TabView Header ItemStyle Selector
+
+Through the `HeaderItemStyleSelector` property you can apply different styles to each TabView header item. The target type of the Style must be `TabViewHeaderItem`.
+
+**1.** Define the TabView control:
+
+<snippet id='tabview-styling-headeritemstyleselector' />
+
+**2.** Define the `StylSelector` resources:
+
+<snippet id='tabview-styling-styleselector-resources' />
+
+**3.** Define the `StylSelector` class:
+
+<snippet id='tabview-header-styleselector-class' />
+
+Here is the TabView after applying the `HeaderItemStyleSelector`:
```

Comment:
See the comment for this sentence in `controls/tabview/styling/header-styling.md`.